aeneas/src/ssa/VstSsaGen.v3

```diff
@@ -353,17 +356,17 @@ class VstSsaGen extends VstVisitor<VstSsaEnv, SsaInstr> {
 		loopEnv.loopStart = loopUpdate;
 		loopEnv.loopEnd = loopEnd;
 		// add the loop control expression
-		var cond = loopEnv.at(null).opIntLt(loopEnv[stmt.loopVar], limitExpr);
+		var cond = loopEnv.opIntLt(loopEnv[stmt.loopVar], limitExpr);
```

Comment:
Can you factor out the `env.at(null)` changes to a separate PR? (Just to keep this PR smaller).